dashboard/config/scripts/starwarsblocks.script

```diff
@@ -0,0 +1,32 @@
+title 'Star Wars: Building a Galaxy with Code'
```

Comment:
awesome.  worth testing as well?  there are some tests covered in https://github.com/code-dot-org/code-dot-org/commit/32dda69212ec987cfc2f099fefbd89c410618a72 though with fixtures not so much is being tested any more.